src/renderers/shared/fiber/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -88,7 +88,7 @@ describe('ReactIncrementalErrorHandling', () => {
     expect(ReactNoop.getChildren()).toEqual([]);
 
     ops.length = 0;
-    ReactNoop.flushDeferredPri(30 + 5 + 5 + 5);
+    ReactNoop.flushDeferredPri(30 + 5 + 5 + 5 + 5);
```

Comment:
I can't remember exactly... it has something to do with the changes I made to the work loop. But I'm changing this completely right now per our discussion. I'll see if I can revert this change, and if not I'll explain why.